Assets/Mirror/Runtime/Transport/Websocket/Server.cs

```diff
@@ -272,7 +276,20 @@ public async void Send(int connectionId, byte[] data)
             {
                 try
                 {
-                    await client.SendAsync(new ArraySegment<byte>(data), WebSocketMessageType.Binary, true, cancellation.Token);
+                    // workaround for: https://forum.unity.com/threads/unity-2017-1-tls-1-2-still-not-working-with-net-4-6.487415/
+                    // In SslStream, only one SendAsync can be going at a time
+                    // if Send is called multiple time, only the first one calls SendAsync,
+                    // the other ones queue up the message
+                    messagesToSend.Enqueue(new ArraySegment<byte>(data));
+                    if (!sendingMessage)
+                    {
+                        sendingMessage = true;
+                        while (messagesToSend.Count > 0)
+                        {
+                            await client.SendAsync(messagesToSend.Dequeue(), WebSocketMessageType.Binary, true, cancellation.Token);
```

Comment:
What happens if SendAsync throws exception?  you will stop sending messages to all clients.